server/api/models/rpaTaskModel.js

```diff
@@ -11,9 +11,9 @@ const rpaParameterSchema = new Schema({
 });
 
 const rpaTaskSchema = new Schema({
-  Application: { type: String, required: [true, 'Application required'] },
-  Task: { type: String, required: [true, 'Task required'] },
-  Code: { type: String, required: [true, 'Code required'] },
+  application: { type: String, required: [true, 'Application required'] },
```

Comment:
Will this still work with the existing data in the database?

Reply:
I will change all the data in the database 😁